cpp/velox/compute/VeloxPlanConverter.cc

```diff
@@ -530,8 +530,19 @@ class VeloxPlanConverter::WholeStageResIterFirstStage
       std::vector<std::shared_ptr<ConnectorSplit>> connectorSplits;
       connectorSplits.reserve(paths.size());
       for (int idx = 0; idx < paths.size(); idx++) {
+        auto partitionInfo = extractPartitionColumnAndValue(paths[idx]);
+        std::unordered_map<std::string, std::optional<std::string>>
```

Comment:
partitionKeys should be out of for loop, is that?

Reply:
Actually the map is only for a single spilt, and split path is sth like below:
```
/mnt/DP_disk1/tpcds_parquet_part_sf100/catalog_returns/cr_returned_date_sk=2451190/part-00120-f24ce1af-4ef3-44d4-9432-c996c5d48970.c000.snappy.parquet
```
What I'm doing here is to extract the partition column and value from `cr_returned_date_sk=2451190` and let the hive connector split know its partition column/value.

Reply:
ok